### DISCUSSION TITLE
Shark Badge

### DISCUSSION THREAD

**Discussion by WaleedRazaShoaib** [community/community#103285] [Discussions] [answered]
### Select Topic Area

Question

### Body

Is it normal that I don't have the pull request badge on my profile? I have a pull request which was merged (on a repository which is not mine, I just contributed with fork), but I don't have the badge. Is there any cron which runs every 2/3 months to create badges, maybe?

**Comment by ahmedrazabaloch** [comment, marked as answer]
You can get Pull Shark Achievement by sumbit two accepted pull requests.
you can use this course to learn, how to get Pull Shark Achievement Step-by-Step by image.
- hope its helpful for you & you get Pull Shark badge as soon as possible

**Reply by WaleedRazaShoaib** [reply]
thanks @ahmedrazabaloch for help

**Comment by queenofcorgis** [comment]
This post has been closed and locked because of version of it has already been posted. We encourage you to use our search bar before posting to find if a similar discussion exists so we don't overwhelm the community with duplicate discussions. Thanks for understanding!

The Community Discussions Team
